### ISSUE TITLE
[feature request] Highlight arrows better

### ISSUE THREAD

**Issue by vmarkovtsev** [google/perfetto#523]
The following screenshot shows the selected arrow event:
![image](https://github.com/google/perfetto/assets/2793551/7ba10afa-102a-44c4-9a21-adc4735a6e78)

It's really hard for me to understand that the dark-green arrow is slightly bigger than the surrounding arrows and is therefore selected. I have a few suggestions for better selection indication:

- Change the arrow icon to something different.
- Draw a thick bevel around the arrow.
- Dynamically flash the arrow like the cursor in a console terminal.

**Comment by stevegolton** [comment]
Hey, thanks for reporting this issue.

I think the problem is not that the selected instant event is drawn slightly larger than the rest, it's that it is drawn with a black outline, which is difficult to distinguish when the randomly selected color is quite dark.

Talking to your suggestions:
 - Change the arrow icon to something different.
I think the chevron works pretty well here, and I can't imagine a different shape being any better, but I'm open to ideas!

 - Draw a thick bevel around the arrow.
What do you mean by bevel? Like a 3d effect?

 - Dynamically flash the arrow like the cursor in a console terminal.
I could be on board with some kind of animating effect. I think flashing might be a bit jarring.

Throwing another idea into the mix - we could colour the border as the opposite colour the center of the arrow to make sure it contrasts well.

I'll look into some of these suggestions.

**Comment by vmarkovtsev** [comment]
Yeah, I mean thick contrast contour by the bevel.

Maybe play some animation only once when the arrow gets selected?

Thanks for considering this, I don't see well so it's really a big deal to me :disappointed:

**Comment by stevegolton** [comment]
Hi @vmarkovtsev.

Apologies, I haven't been able to spend much time on this, unfortunately t's a low priority.

We are planning on revamping selection in the future, but as a temporary fix I've updated the selection to use a bounding box, as is used elsewhere in the UI. I think this is easier to see than the outline approach.

Before:
![image](https://github.com/google/perfetto/assets/6147146/501f8617-a0a9-4c57-b6dd-c7cf71f90f6c)

After:
![image](https://github.com/google/perfetto/assets/6147146/212b040a-fccb-4e70-a579-32d2560a00c7)

Let me know what you think. I can always revert the change, but I'm unlikely to be able to schedule much more time on this in the near future.

**Comment by vmarkovtsev** [comment]
OK, that should be helpful. I cannot test it short-term, alas.

**Comment by stevegolton** [comment]
Great! I'll tentatively close this issue but feel free to re-open.
